README.md

```diff
@@ -51,13 +53,140 @@ To run this project do the following:
+
+2. Double-click the installer package to mount the Docker disk image.
+
+3. Drag the Docker icon to the Applications folder.
+
+4. Open Docker from the Applications folder and follow the on-screen instructions to complete the installation.
+
+5. Once the installation is complete, Docker Desktop will be running on your system.
+
+#### Install Docker on Linux
+NB: Refer to the official Docker documentation for updates: <a href="https://docs.docker.com/engine/install/ubuntu/">Get Docker for Linux.</a>
+
+1. Open a terminal window.
+
+2. Update the package index:<br>
+    ```sudo apt-get update```
+
+3. Install Docker dependencies:<br>
+    ```sudo apt-get install apt-transport-https ca-certificates curl software-properties-common```
+
+4. Add the Docker GPG key:<br>
+    ```curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg```
+
+5. Set up the stable Docker repository:<br>
+    ```echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null```
+
+6. Update the package index again:<br>
+    ```sudo apt-get update```
+
+7. Install Docker:<br>
+```sudo apt-get install docker-ce docker-ce-cli containerd.io```
+
+Docker should now be installed on your Linux system.
+
+
+
+Check if Docker is installed correctly:<br>
+    ```docker --version```
+
+#### Run the application
+To run this project using docker, please follow these steps:
+
+1. Change to the repository directory on your computer (if you are not already there):<br>
+    ```cd chitchat```<br>
+    In this directory you should see a file:<br>
+     - docker-compose.yaml
+
+2. Then in your terminal, run the following command:<br>
+    ```docker compose up --build```
+
+This should bring up the entire project. Good luck!
+
 
 ## Contributing
 
-Pull requests are welcome. For major changes, please open an issue first
-to discuss what you would like to change.
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change. Please make sure to update tests as appropriate. This section is a simplified guide to make contributions. Follow the steps below to contribute:
+
+### Fork the repository
+
+Fork this repository by clicking on the fork button on the top of this page. This will create a copy of this repository in your account.<br>
+    <img src="readme/fork-screenshot.png" alt="Fork screenshot" width="400" height="120">
+
+### Clone the repository
+
+The next step is to clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the copy to clipboard icon. The copied url should look like this: https://github.com/PythonBulawayo/ChitChat.git<br>
```

Comment:
@Awakhiwekyle In addition to HTTPS, you can clone the repo using SSH. Let's add a note here on how to clone using an SSH github URL if the contributor prefers using SSH over HTTPS:


`git@github.com:PythonBulawayo/ChitChat.git`

Reply:
Noted @terrameijar, thanks.